configuration/enable-offline-mode.adoc

```diff
@@ -46,12 +46,12 @@ include::modules/enable-offline-mode-roxctl.adoc[leveloffset=+2]
 [id="update-scanner-definitions-in-offline-mode"]
 == Updating Scanner definitions in offline mode
 
-Scanner contains a local vulnerability definitions database. When {product-title} runs in normal mode (connected to the internet), Scanner fetches new vulnerability definitions from the internet and updates its database.
+Scanner contains a local vulnerability definitions database. If {rh-rhacs-first} is running in normal mode with an internet connection, Scanner fetches new vulnerability definitions from the internet and updates its database.
 //TODO: Add link to Vulnerability definitions
 
-However, when you are using {product-title} in offline mode, you must manually update Scanner definitions by uploading them to Central.
+However, when you are using {product-title-short} in offline mode, you must manually update Scanner definitions by uploading them to Central.
 
-When {product-title} runs in offline mode, Scanner checks for new definitions from Central. If new definitions are available, Scanner downloads the new definitions from Central, marks them as default, and then uses the updated definitions for scanning images.
+If {product-title-short} is running in offline mode, Scanner checks every 3 hours whether new definitions are available from Central. If new definitions are available, Scanner downloads the new definitions from Central, marks them as default and then uses the updated definitions to scan images.
```

Comment:
> If {product-title-short} is running in offline mode, Scanner checks every 3 hours whether new definitions are available from Central.

That's not quite true. Scanner checks with Central every ~5 minutes (plus some jitter). Central checks if there is new data also every ~5 minutes + some jitter. The data is updated elsewhere ~every 3 hours (though this is very subject to change)